x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart_annotations.ts

```diff
@@ -148,18 +162,9 @@ export function renderAnnotations(
     .attr('ry', ANNOTATION_RECT_BORDER_RADIUS)
     .classed('mlAnnotationRect', true)
     .attr('mask', `url(#${ANNOTATION_MASK_ID})`)
-    .on('mouseover', function (this: object, d: Annotation) {
-      showFocusChartTooltip(d, this);
-    })
-    .on('mouseout', () => hideFocusChartTooltip())
-    .on('click', (d: Annotation) => {
-      // clear a possible existing annotation set up for editing before setting the new one.
-      // this needs to be done explicitly here because a new annotation created using the brush tool
-      // could still be present in the chart.
-      annotationUpdatesService.setValue(null);
-      // set the actual annotation and trigger the flyout
-      annotationUpdatesService.setValue(d);
-    });
+    .on('mouseover', onAnnotationMouseOver)
+    .on('mouseout', onAnnotationMouseOut)
```

Comment:
Can't `hideFocusChartTooltip` be passed here directly since it takes no arguments? Does it need to be wrapped in an arrow function?

Reply:
Thanks for catching that. Updated here [`dc9798b` (#98233)](https://github.com/elastic/kibana/pull/98233/commits/dc9798b6be5dbe6bd4db0ca5a53132edfe4b8c9e).